src/ui/buttons/props.js

```diff
@@ -696,6 +719,56 @@ export function normalizeButtonStyle(
   };
 }
 
+export function validateButtonMessage(
+  props: ?ButtonPropsInputs,
+  message: ButtonMessageInputs
+): ButtonMessage {
+  if (!message) {
+    throw new Error(`Expected props.message to be set`);
+  }
+
+  props = props || getDefaultButtonPropsInput();
+
+  const { amount, offer, color, position, align } = message;
+
+  if (amount !== undefined) {
```

Comment:
It's generally considered best practice to use `typeof amount !== "undefined"` due to old JS shenanigans.

Reply:
noted, will fix